fast64_internal/oot/oot_level_writer.py

```diff
@@ -673,7 +680,7 @@ def ootRemoveSceneC(exportInfo):
 
 class OOT_RemoveScene(bpy.types.Operator):
     bl_idname = "object.oot_remove_level"
-    bl_label = "Remove Scene"
+    bl_label = "Are you sure you want to remove this level?"
```

Comment:
```suggestion
    bl_label = "Are you sure you want to remove this scene?"
```
("level" isn't really used in oot)

A better way to do this may be to implement operator.draw and use a layout.label(text=) to print this warning, bl_label isn't really intended for this
For example this message shows up in operator search
![image](https://user-images.githubusercontent.com/5306619/192163819-779b9446-c0ef-4c29-83dd-b3d04e5a5795.png)

Reply:
That's true about the bl_label, the main reason I did that was because I thought there wasn't any other way to get a confirmation dialog with a message. But I found out I could use invoke_props_dialog() which uses the draw function instead of invoke_confirm(). I changed the text as well.